### PR TITLE
ticdc: fix kafka message size limit

### PR DESCRIPTION
**Pull request by amyangfei**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

TiCDC provides two parameters to control the message size and batch size of Kafka messages.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by amyangfei** [comment]
The CI failure is not related to change in this PR
```
Link checker output file: link-checker/out.md
tiup/tiup-dm-topology-reference.md
	ERROR	https://github.com/pingcap/tiup/blob/master/examples/dm/topology.example.yaml
		Not Found (HTTP error 404)
```

**Comment by TomShawn** [comment]
> The CI failure is not related to change in this PR
> 
> ```
> Link checker output file: link-checker/out.md
> tiup/tiup-dm-topology-reference.md
> 	ERROR	https://github.com/pingcap/tiup/blob/master/examples/dm/topology.example.yaml
> 		Not Found (HTTP error 404)
> ```

There's nothing wrong in this PR about it. That will be fixed in a separate PR, so let's ignore it here.

File: ticdc/troubleshoot-ticdc.md [hunk 1]
```diff
@@ -240,7 +240,8 @@ cdc cli changefeed query --pd=http://10.0.10.25:2379 --changefeed-id 28c43ffc-23
 
 ## TiCDC 把数据同步到 Kafka 时，能在 TiDB 中控制单条消息大小的上限吗？
 
-不能，目前 TiCDC 控制了向 Kafka 发送的消息批量的大小最大为 512 MB，其中单个消息的大小最大为 4 MB。
+* 从 4.0.9 开始，可以通过 `max-message-bytes` 控制每次向 Kafka broker 发送消息的最大数据量（可选，默认值 64MB）；通过 `max-batch-size` 参数指定一条 kafka 消息中变更记录的最多数量，目前仅对 Kafka 的 protocol 为 default 时有效（可选，默认值为 4096）;
+* 4.0.9 之前，TiCDC 向 Kafka 发送的消息批量的大小最大为 512 MB。
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:244]: We can add this back in release-4.0.
```suggestion
可以通过 `max-message-bytes` 控制每次向 Kafka broker 发送消息的最大数据量（可选，默认值 64MB）；通过 `max-batch-size` 参数指定每条 kafka 消息中变更记录的最大数量，目前仅对 Kafka 的 protocol 为 `default` 时有效（可选，默认值为 `4096`）;
```

**Pull request merged**